src/platform/types.js

```diff
@@ -48,53 +53,4 @@ export type PlatformApi = {
   fetchManifest: () => Promise<AppManifest[]>,
 };
 
-export type PlatformAccount = {
-  id: string,
-  name: string,
-  address: string,
-  currency: string,
-  balance: BigNumber,
-  spendableBalance: BigNumber,
-  blockHeight: number,
-  lastSyncDate: Date,
-};
-
-export type PlatformUnit = {
-  name: string,
-  code: string,
-  magnitude: number,
-};
-
-export type PlatformCurrency = {
-  type: string,
-  color: string,
-  ticker: string,
-  id: string,
-  name: string,
-  family: string,
-  units: PlatformUnit[],
-};
-
-export interface PlatformTransactionCommon {
-  amount: BigNumber;
-  recipient: string;
-}
-
-export interface PlatformEthereumTransaction extends PlatformTransactionCommon {
-  family: "ethereum";
-  nonce: ?number;
-  data: ?Buffer;
-  gasPrice: ?BigNumber;
-  gasLimit: ?BigNumber;
-}
-
-export interface PlatformBitcoinTransaction extends PlatformTransactionCommon {
-  family: "bitcoin";
-  feePerByte: ?BigNumber;
-}
-
-export type PlatformTransaction =
-  | PlatformEthereumTransaction
-  | PlatformBitcoinTransaction;
-
 export type PlatformSignedTransaction = SignedOperation;
```

Comment:
This type should also be implemented in the SDK, but since it depends on a more complex operation type this has not been done today. cf. https://github.com/LedgerHQ/ledger-live-platform-sdk/blob/main/src/types.ts#L15-L22

Regarding this operation type, we will need to find a clever way to avoid duplication between LLC and LLP-SDK (maybe extracting it on a separate package / repo from which LLC and LLP-SDK would depend 🤔 ).
This could also make sense if some other project, present or future (maybe Vault) also use a quite similar operation type

Reply:
Here is a proposed solution to decouple operation from LLP-SDK

tldr; SDK is operation agnostic and all operation related logic (highly correlated to Live specific logic) stays in LLC.

https://github.com/LedgerHQ/live-app-sdk/pull/20